services/madoc-ts/src/frontend/shared/components/SearchResults.tsx

```diff
@@ -90,7 +101,7 @@ const SearchItem: React.FC<{ result: SearchResult; size?: 'large' | 'small'; sea
   const isManifest = result.resource_type === 'Manifest';
 
   return (
-    <ResultContainer>
+    <div>
```

Comment:
Should the CSS of `<ResultContainer />` be changed instead?

Reply:
Changed to fragment. ResultContainer is down under loading block